np16.py

```diff
@@ -0,0 +1,20 @@
+import numpy as np
+
+
+def sort_student(student_id, student_height):
+    """
+    Function that sort the student id with increasing height
+    Print the integer indices that
+    describes the sort order by multiple columns and the sorted data
+    """
+    sorted_indexes = np.lexsort((student_id, student_height))
+    print(sorted_indexes)
+    print('sorted data:')
```

Comment:
Seperate IO from logic -
 don’t use print in functions unless there’s a good reason